### DISCUSSION TITLE
Impossible to download from Linkedin. Compagnon app not recognized !!!

### DISCUSSION THREAD

**Discussion by Albatros78** [aclap-dev/video-downloadhelper#932] [Help and Q&A]
### Your question

Hell all,

Here the recommandations from the website and my answers

If the app is not recognized:

The browser you're using (be as specific as possible)
---- **Chrome Version 109.0.5414.120 (Build officiel) (64 bits)**

OS (Linux/Windows/Mac) & OS version
----  **Windows 7 SP1**

OS 64bits or 32bits
----  **64 bits**

Link to the coapp you installed
----  **C:\Program Files\DownloadHelper CoApp**

On Mac, the output of /Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp --info
On Windows, open a CMD shell, go to the install directory, and type vdhcoapp.exe --info
---- **C:\Program Files\DownloadHelper CoApp>vdhcoapp.exe --info
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "win7",
    "arch": "x86_64",
    "node": "10"
  },
  "home": "C:\\Users\\Axel",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f36095"
}**

If the download video is not valid or missing audio:

version of Video DownloadHelper addon
---- **v9.0.2.12**

link to the page with the video
---- **Any video on linkedin network**

Previously it was OK. I have reinstalled from an OS image on my PC and now the reinstallation fails... (Already done 2 or 3 times )
I have added all exe files like exceptions from my Anti Virus (Avast)

Any advice  !!!!

Thank you

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: fr
coapp: {"error":"Failed to start native messaging host.","found":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36
{
    "used_sidebar_button": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 12
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I can't find "any" video on linkedin.  If you give me the URL of some specific page on linkedin where there is a problematic video, or a few such URLs, I could investigate your problem.  However, if I have to log into linkedin, I won't be able to help because I am not a member of that site & intend to never become a member.

For your "native messaging host" error, there's been some discussions in here on that topic that might be helpful.  You should try to find them & read them.

**Reply by Albatros78** [reply]
Hello,

Thank you for your reply.
I tried to find some posts with "native messaging error" string but nothing found !!!

What kind of string have you used for searching into posts ?

rgds

**Reply by GrampaWildWilly** [reply]
Try just "native messaging" as your search key.

Also, about some URLs . . . ?

**Reply by Albatros78** [reply]
Hello, I found that this issuz was with Linkedin videos... It works with some other videos (not youtube) on some websites.

If you have a linkedin account you can try directly... problem is with ALL videos  and not just only few.
And I cannot gice to you Linkedin URL's because it's with my private account

Any advice, why does not works with Linkedin... ?

And with Facebook, impossible to download video. Just my profile picture is displayed !

![image](https://github.com/aclap-dev/video-downloadhelper/assets/79452305/6424f024-a659-46c5-bf93-cbb21a2006f7)

**Reply by Mattmjs** [reply]
Is there a public linkedin video you can share here or are they private ? What was the issue ?
Is there detection or failures ?

**Reply by Albatros78** [reply]
Linkedin needs to have an account.

Here when I ty to download a video

![image](https://github.com/aclap-dev/video-downloadhelper/assets/79452305/3b8a673d-c4ca-4e5e-8ded-46478a889e41)

Always same error where the app is not installed... (2 or 3 installations done !!!)

And video sharing is not possible (no option) on Linkedin !!

**Reply by Albatros78** [reply]
Seems really not possible on likedin to geta video URL or to use the right click... Nothing works !!!

So maybe VDH needs to work on it, I believe.
